packages/arb-token-bridge-ui/src/components/TransactionHistory/PendingTransactions.tsx

```diff
@@ -26,9 +26,7 @@ export const PendingTransactions = ({
     <div className="relative flex max-h-[500px] flex-col gap-4 overflow-auto rounded-lg bg-blue-arbitrum p-4">
       {/* Heading */}
       <span className="flex flex-nowrap items-center gap-x-3 whitespace-nowrap text-xl text-white">
-        {loading && (
-          <Loader type="TailSpin" color="white" width={20} height={20} />
-        )}
+        {loading && <Loader color="white" size="small" />}
```

Comment:
are we sure we want to change from 20px to 16px?

Reply:
Visually it looks okay with the Pending Transaction text. The design for this element wasn't documented. Wdyt?
<img width="300" alt="image" src="https://user-images.githubusercontent.com/7558499/224981757-7e0182cb-75d7-4902-b380-f420be1fb196.png">